CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1223,15 +1223,36 @@ end
 --| End Gamepad Functions |--
 -----------------------------
 
+local function EvaluateBackpackPanelVisibility(enabled)
+	if not enabled then
+		return false
+	end
+	if not StarterGui:GetCoreGuiEnabled(Enum.CoreGuiType.Backpack) then
+		return false
+	end
+	if not TopbarEnabled then
+		return false
+	end
+	if not UserInputService.VREnabled then
+		return false
+	end
+
+	return true
```

Comment:
Isn't this whole function the same as:
` return enabled and StarterGui:GetCoreGuiEnabled(Enum.CoreGuiType.Backpack) and TopbarEnabled and UserInputService.VREnabled`